Source/control.cpp

```diff
@@ -1026,6 +1083,27 @@ void DrawInfoBox(const Surface &out)
 			AddPanelString(fmt::format(fmt::runtime(_("{:s}, Level: {:d}")), _(ClassStrTbl[static_cast<std::size_t>(target._pClass)]), target._pLevel));
 			AddPanelString(fmt::format(fmt::runtime(_("Hit Points {:d} of {:d}")), target._pHitPoints >> 6, target._pMaxHP >> 6));
 		}
+		if (sbookflag) {
+			Player &player = *MyPlayer;
+			uint64_t spl = player._pMemSpells | player._pISpells | player._pAblSpells;
+			const int spellSlots = 7;
+			spell_id snPos[spellSlots];
+			Rectangle iconAreas[spellSlots];
+
+			for (int i = 0; i < spellSlots; i++) {
+				spell_id sn = SpellPages[sbooktab][i];
+				snPos[i] = sn;
+				iconAreas[i] = { GetPanelPosition(UiPanels::Spell, { 11, 18 + (SpellBookDescription.height * i) }), Size { 37, SpellBookDescription.height - 5 } };
+			}
+			int panelWidth = 240;

```

Comment:
Doesn't look like this variable is ever modified? In this case, consider:

```suggestion
			constexpr int PanelWidth = 240;
```